pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -665,12 +698,21 @@ func buildDefaultPassthroughCluster() *v2.Cluster {
 	return cluster
 }
 
+// TODO: supply LbEndpoints or even better, LocalityLbEndpoints here
+// change all other callsites accordingly
 func buildDefaultCluster(env *model.Environment, name string, discoveryType v2.Cluster_DiscoveryType,
-	hosts []*core.Address) *v2.Cluster {
+	lbEndpoints []endpoint.LbEndpoint) *v2.Cluster {
```

Comment:
looking at this code, do you think its useful to supply localityLbEndpoints instead of plain LbEndpoints? In near future, when we implement locality aware routing, these functions wont change. We will change the implementation of the buildClusterLocalityLbEndpoints to include the locality info

Reply:
To support `locality aware routing`, It's better to build localityLbEndpoints now.